### PR TITLE
Included information on vra 7.5

### LINKED ISSUE #1894: Document VIC requirements for vRA integration
Per email thread started by @dbarkelew.

### PR DESCRIPTION
**Pull request by vidya-v**
Added support information for vRA 7.5.
@stuclem , can you please review this? Thanks!

Fixes #1894

### CODE REVIEW

File: docs/user_doc/vic_vsphere_admin/interop.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ This topic describes how vSphere features and other VMware products interact wit
...
 
-**NOTE**: vRealize Automation 7.3 supports vSphere Integrated Containers 1.1.x. vRealize Automation 7.4 supports vSphere Integrated Containers 1.3.x. Support for this release of vSphere Integrated Containers will be added in a future release of vRealize Automation. For the most up-to-date information about compatibility with vRealize Automation, see the [VMware Product Interoperability Matrices](https://partnerweb.vmware.com/comp_guide2/sim/interop_matrix.php#interop&114=&149=).
+**NOTE**: vRealize Automation 7.3 supports vSphere Integrated Containers 1.1.x. vRealize Automation 7.4 supports vSphere Integrated Containers 1.3.x. vRealize Automation 7.5 supports vSphere Integrated Containers 1.3.x and 1.4.x, up to and including 1.4.2. For the most up-to-date information about compatibility with vRealize Automation, see the [VMware Product Interoperability Matrices](https://partnerweb.vmware.com/comp_guide2/sim/interop_matrix.php#interop&114=&149=).
```
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/interop.md]: Maybe make a bulletted list for the 3 versions of vRA?

Comment (vidya-v) [reply docs/user_doc/vic_vsphere_admin/interop.md]: Thanks, @stuclem . I have made the changes.

**Comment by martin-borisov** [comment]
Looks good.

**Comment by stuclem** [comment]
LGTM!

**Pull request merged**